src/.clang-tidy

```diff
@@ -0,0 +1,28 @@
+---
+Checks:          '-*,
+                 # modernize-use-auto,
+                 # modernize-use-equals-delete,
+                 # modernize-deprecated-headers alreday applied, but stuck with https://github.com/nodejs/node/pull/26159#issuecomment-464417206
```

Comment:
I revert `js_native_*` because some test failed back then.

Reply:
You can add
```c
// NOLINTNEXTLINE(modernize-deprecated-headers)
```
https://clang.llvm.org/extra/clang-tidy/#suppressing-undesired-diagnostics

But anyway, the job I've created (https://ci.nodejs.org/job/node-clang-tidy/) only runs for `*.cc` files.

Reply:
PR: https://github.com/nodejs/node/pull/26884